File: C/ATIVIDADE-10/ATIVIDADE-1.c
/*
01. Faça um programa que permita entrar com o nome, a nota da prova 1 e a nota da prova 2 
de um aluno. O programa deve imprimir o nome, a nota da prova 1, a nota da prova 2, a média
das notas e uma das mensagens: "Aprovado", "Reprovado" ou "em Prova Final"(a média é 7 para 
aprovação, menor que 3 para reprovação e as demais em prova final).
*/

#include <stdio.h>
#include <stdlib.h>
#include <locale.h>


int main()
{
    setlocale(LC_ALL,"Portuguese");

   float num1, num2, media;

   printf("Digite a primeira nota: ");
   scanf("%f",&num1);

   printf("Digite a segunda nota: ");
   scanf("%f",&num2);

   media = (num1 + num2) / 2;

   if (media <= 3){
    printf("NOTA 1: %.1f\n",num1);
    printf("NOTA 2: %.1f\n",num2);
    printf("MEDIA: %.1f\n",media);
    printf("O aluno está Reprovado\n");
   } else if(media <= 7){
    printf("NOTA 1: %.1f\n",num1);
    printf("NOTA 2: %.1f\n",num2);
    printf("MEDIA: %.1f",media);
    printf("O aluno está Recuperação\n");
   } else{
    printf("NOTA 1: %.1f\n",num1);
    printf("NOTA 2: %.1f\n",num2);
    printf("MEDIA: %.21\n",media);
    printf("O aluno está APROVADO\n");
   }

}

File: JAVA/ATIVIDADE-4/ATIVIDADE1.java
import java.util.Scanner;

/*01. Crie um programa em linguagem Java que permita entrar com a nota da prova 1 e da prova 2 de 15 alunos. Ao final, imprimir uma listagem, contendo: nota da prova 1, nota da prova 2, a média das notas de cada aluno e a situação final de cada. Ao final, imprimir a média geral da turma.
*/
public class ATIVIDADE1 {
    public static void main(String[] args) {

        double nota1,nota2,media;

        Scanner entrada = new Scanner(System.in);

        for(int i = 0; i < 5; i++){
            System.out.print("Digite a primeira nota: ");
            nota1 = entrada.nextDouble();

            System.out.print("Digite a segunda nota: ");
            nota2 = entrada.nextDouble();

            media = (nota1 + nota2)/2;

            System.out.println("======================================================");
            System.out.printf("ALUNO %d\n",i+1);
            System.out.printf("NOTA 1 %.1f\n",nota1);
            System.out.printf("NOTA 2 %.1f\n",nota2);
            System.out.printf("MEDIA = %.2f\n",media);
        }

        entrada.close();
    }
}
